File: ASPNETCore/HowTo/WebApiTypescriptSample/src/Modules/Excel/Dtos/DeleteRow.dto.ts
import { BaseDto } from "../../../Base/Base.Dto";

export class DeleteRowDto extends BaseDto {
    
    /**The excel file name that storage manager can recognize. */
    public ExcelPath: string;

    /**The sheet name.*/
    public SheetName: string;

    /**the row indexes. format: {num}; {num},{num},...; {num}-{num}. for example: 1,3-5 */
    public RowIndexs: string;

}

File: ASPNETCore/HowTo/WebApiConsoleSample/src/Modules/Excel/Dtos/DeleteRow.dto.cs
namespace WebApiConsoleSample
{
    public class DeleteRowDto : BaseDto
    {
        /**The excel file name that storage manager can recognize. */
        public string ExcelPath { set; get; }

        /**The sheet name.*/
        public string SheetName { set; get; }

        /**the row indexes. format: {num}; {num},{num},...; {num}-{num}. for example: 1,3-5 */
        public string RowIndexs { set; get; }

    }
}